modules/HapticGlove_module/src/GloveControlHelper.cpp

```diff
@@ -199,8 +200,14 @@ bool GloveControlHelper::setFingertipVibrotactileFeedbackReferences(
             = (int)std::round(std::max(0.0, std::min(desiredValue[i], 100.0)));
     }
 
-    return m_pImp->setFingertipVibrotactileValues(m_desiredVibrotactileValues);
+    return true;
 }
+
+bool GloveControlHelper::setFingertipHapticFeedbackReferences()
```

Comment:
Probably we could use the keyword ``send`` instead of ``set`` to indicate that these values will be "sent" to the user

Reply:
Yes, I agree. Changed to `send`